scripts/build_bus_regions.py

```diff
@@ -145,7 +147,35 @@ def custom_voronoi_partition_pts(points,
     polygons_arr[:] = polygons
     return polygons_arr
 
-
+def get_gadm_shape(onshore_locs):
+    def locate_bus(coords):
+        try:
+            return gadm_shapes[gadm_shapes.contains(Point(coords['x'],
```

Comment:
gadm_shapes may be given as an argument in the function, and when the function is called, we can give them the proper argument
This may be more readable and manutentable in my opinion; do you agree?